README.rst

```diff
@@ -54,11 +54,20 @@ save it as a file called ``config.json``:
         "page_id": "YOUR_FB_PAGE_ID",
         "retrieve_profile": false,
         "outbound_url": "https://graph.facebook.com/v2.6/me/messages",
```

Comment:
Please change version from 2.6 to 2.8, as that is the minimum version for new apps (and I tested it, it doesn't break stuff)